paramak/parametric_components/tokamak_plasma.py

```diff
@@ -183,8 +183,7 @@ def Z(theta):
         points = np.stack((R_points, Z_points), axis=1)
 
         # set self.points
-        self.points = list(points)
-
+        self.points = [(p[0], p[1]) for p in points[:-1]]
```

Comment:
any chance you could add a comment to say why all the points are not used

Reply:
I added a comment. This is related with the PR i opened a while ago : if points[0] == points[-1] and error is raised (even though in the end, points[0] is appended at the end of points...). I still don't think an error should be raised here